core-httpclient-impl/src/main/java/com/optimizely/ab/config/HttpProjectConfigManager.java

```diff
@@ -357,7 +365,8 @@ public HttpProjectConfigManager build(boolean defer) {
                 datafileAccessToken,
                 blockingTimeoutPeriod,
                 blockingTimeoutUnit,
-                notificationCenter);
+                notificationCenter,
+                sdkKey);
```

Comment:
For existing clients using ".withUrl()" method, `sdkKey` can be null here.
What about moving lines 345-347 to 343, so we clarify a breaking change to those clients, instead of letting them debug why ODP not working?